components/BaseMinMaxSlider.vue

```diff
@@ -0,0 +1,238 @@
+        :max="max"
+        :value="maxValue"
+        :step="step"
+        @input="onInput"
+      />
+    </div>
+    <div class="slider__inputs">
+      <div class="slider__input">
+        <BaseInput
+          v-model="sliderMinValue"
+          name="min-size"
+          type="number"
+          :step="step"
+          :min="min"
+          :max="max"
+        >
+          <span v-if="unit" class="slider__input--unit">{{ unit }}</span>
+        </BaseInput>
+      </div>
+      <div class="minmax__dash"></div>
+      <div class="slider__input">
+        <BaseInput
+          v-model="sliderMaxValue"
+          name="max-size"
+          type="number"
+          :step="step"
+          :min="min"
+          :max="max"
+        >
+          <span v-if="unit" class="slider__input--unit">{{ unit }}</span>
+        </BaseInput>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.slider__label {
+  padding-bottom: 0.75rem;
+}
+.slider__range {
+  --trackHeight: 0.5rem;
+  --thumbRadius: 1rem;
+}
+
+/* style for the input element with type "range" */
+.slider__range input[type="range"] {
+  position: relative;
+  appearance: none;
+  background: none;
+  pointer-events: none;
+  border-radius: 999px;
+  z-index: 0;
+}
+
+/* ::before element to replace the slider track */
+.slider__range input[type="range"]::before,
+.slider__range.minmax::before {
+  content: "";
+  position: absolute;
+  width: var(--ProgressPercent, 100%);
+  height: 100%;
+  background: $color-green-dark;
+  pointer-events: none;
+  border-radius: 999px;
+}
+
+.slider__range input[type="range"]::-webkit-slider-thumb {
+  position: relative;
+  width: var(--thumbRadius);
+  height: var(--thumbRadius);
+  margin-top: calc((var(--trackHeight) - var(--thumbRadius)) / 2);
+  background: $color-green-dark;
+  border-radius: 999px;
+  pointer-events: all;
+  appearance: none;
+  z-index: 1;
+}
+
+.slider__range.minmax {
+  position: relative;
+  height: var(--trackHeight);
+  background: $color-green-medium;
+  border-radius: 999px;
+  margin: 0.5rem 0;
+  --progressLeft: 0%;
+  --progressRight: 0%;
+}
+
+.slider__range.minmax input[type="range"] {
+  position: absolute;
+  pointer-events: none;
+  width: 100%;
+}
+
+.slider__range.minmax input[type="range"]::-webkit-slider-runnable-track {
+  background: none;
+}
+
+.slider__range.minmax::before {
+  left: var(--progressLeft);
+  right: var(--progressRight);
+  width: unset;
+}
+
+.slider__range.minmax input[type="range"]::before {
+  display: none;
+}
+
+.slider__inputs {
+  padding-top: 0.5rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.minmax__dash {
+  width: 15px;
+  height: 2px;
+  border-radius: 8px;
+  background-color: $text-color-secondary;
+  @include smallScreen {
+    width: 30px;
+  }
+}
+
+.slider__input--unit {
+  padding: 0.5rem 0 0.5rem 0.5rem;
+}
+
+.slider__input {
+  width: 100px;
+  @include smallScreen {
+    width: 150px;
+  }
+}
```

Comment:
generally I use scss variables, here are some css variables and honestly I would leave it like this cuz I think it is the first time I see that usage of css variables like here

Reply:
I removed the state you mentioned and I had to replace BaseInput with input tag. Generally I refactored the component so that it is much simpler, maybe some repetition in computed returning the percent - but I don't have to watch them anymore.

Reply:
It's not necessary to replace `BaseInput` with `input` 👌 one difference is that you are not able to use `v-model` the same way as before so you can use `:model-value` and `@update:model-value` instead 😄

Reply:
Thank you, I changed to BaseInput and :model-value, but honestly don't understand it and couldn't really find info in the docs :(